docs/root/faq/api/control_plane.rst

```diff
@@ -0,0 +1,28 @@
+How do I support multiple xDS API major versions in my control plane?
+=====================================================================
+
+Where possible, it is highly recommended that control planes support a single major version at a
+given point in time for simplicity. This works in situations where control planes need to only
+support a window of Envoy versions which spans less than a year.
+
+For control planes that need to support a wider range of versions, there are a few approaches:
+
+1. Independent v2/v3 configuration generation pipelines. This is simple to understand but
```

Comment:
nit: vn/vn+1 to be more generic? same else where or one top level sentence that says "assuming current version is v2 and next major version is v3` or some thing like that